daml-foundations/daml-ghc/tests/InternedExternalRefs.daml

```diff
@@ -0,0 +1,8 @@
+-- Copyright (c) 2019, Digital Asset (Switzerland) GmbH and/or its affiliates.
+-- All rights reserved.
+
+-- @ SINCE-LF 1.6
+daml 1.2 module InternedExternalRefs where
+
+-- @ QUERY-LF .modules[] | .values[] | select(.name_with_type.name == ["reverseCopy"]) | .expr.val.module.package_ref.interned_id >= 0
```

Comment:
Does this really test what we want? Isn't `0` the default for numerical values and hence this query always succeeds?

Reply:
Yes; when you disable interning in `EncodeV1` you get

```
      running jq query: .modules[] | .values[] | select(.name_with_type.name == ["reverseCopy"]) | .expr.val.module.package_ref.interned_id >= 0
      jq query failed: got false
```

Reply:
Good to know. Thanks for double checking.